aptoide-views/src/main/java/cm/aptoide/aptoideviews/video/YoutubeWebViewPlayer.kt

```diff
@@ -0,0 +1,103 @@
+package cm.aptoide.aptoideviews.video
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.util.AttributeSet
+import android.webkit.WebResourceError
+import android.webkit.WebResourceRequest
+import android.webkit.WebView
+import android.webkit.WebViewClient
+
+class YoutubeWebViewPlayer : WebView {
+
+  private val youtubeWebViewClient = YoutubeWebViewClient()
+
+  constructor(context: Context) : this(context, null)
+  constructor(context: Context, attrs: AttributeSet?) : this(context, attrs, 0)
+  constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(context, attrs,
+      defStyleAttr) {
+    webViewClient = youtubeWebViewClient
+    getSettings().setJavaScriptEnabled(true)
+  }
+
+  init {
+    setOnTouchListener { v, event ->
+      val removeEndScreen = "document.getElementsByClassName('ytp-endscreen-content')[0].remove();"
+      val removeEndScreenHtml5 = "document.getElementsByClassName('html5-endscreen')[0].remove();"
+      val changeReplayPosition =
+          "document.getElementsByClassName('ytp-replay-button')[0].style.top=0;"
+      val removePauseOverlay =
+          "document.getElementsByClassName('ytp-pause-overlay')[0].remove();"
+      this.evaluateJavascript(
```

Comment:
this method requires api 19, our minimum is 16. Can we do this? Since this is wallet related, will it only be seen by people on devices >= 21?

Reply:
Added API check.